pkg/policy/execute.go

```diff
@@ -2,17 +2,44 @@ package policy
 
 import (
```

Comment:
nit: General rule of thumb we want private functions to be defined after public functions, the order here is not by convention